ui/src/views/Settings.vue

```diff
@@ -324,6 +344,74 @@ export default {
         return;
       }
 
+      // check if nethvoice adm exists
+      var exists = this.users[this.form.user_domain].filter((user) => {
+        return user.user === this.instanceName + "-adm";
+      });
+
+      // create nethvoice adm user, if not exists
+      if (exists.length == 0) {
+        // compose credentials
+        this.form.nethvoice_adm.username = this.instanceName + "-adm";
+        this.form.nethvoice_adm.password = this.generatePassword();
+
+        // execute task
+        const resAdm = await to(
+          this.createModuleTaskForApp(this.providers[this.form.user_domain], {
+            action: "add-user",
+            data: {
+              user: this.form.nethvoice_adm.username,
+              display_name: this.instanceName + " Administrator",
+              password: this.form.nethvoice_adm.password,
+              locked: false,
+              groups: ["domain admins"],
+            },
+            extra: {
+              title: this.$t("settings.create_nethvoice_adm"),
+              description: this.$t("common.processing"),
+              eventId,
+            },
+          })
+        );
+        const errAdm = resAdm[0];
+
+        // check error
+        if (errAdm) {
+          console.error(`error creating task ${taskAction}`, errAdm);
+          this.error.configureModule = this.getErrorMessage(errAdm);
+          this.loading.configureModule = false;
+          return;
+        }
+      } else {
+        // if domain changed
+        if (this.config.user_domain != this.form.user_domain) {
+          // change password
+          const resAdm = await to(
+            this.createModuleTaskForApp(this.providers[this.form.user_domain], {
+              action: "alter-user",
```

Comment:
If we change the user's domain, we cannot be sure that the admin user is present in the new accounts provider.

Reply:
we check before if exists: https://github.com/nethesis/ns8-nethvoice/pull/150/files#diff-21d0072c380dfeb2c3e0eed140d09400099a4fbd448bd9bf7dc7e18960ed585dR353 otherwise we will create it. if already exists we only set new password.